src/lib/components/PostPreview/PostPreviewCard.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { makeStyles, createStyles } from '@material-ui/core';
+import Card from '@material-ui/core/Card';
+import Box from '@material-ui/core/Box';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import { IExpert, IPost } from '../../types';
+import { directionProperties } from './direction-properties';
+import { postTypeProperties } from './post-type-properties';
+
+const useStyles = makeStyles(
+  (theme) =>
+    createStyles({
+      root: {
+        maxWidth: 300,
+        position: 'relative',
+      },
+      media: {
+        height: 58,
+        width: 46,
+        paddingTop: 13,
+        paddingLeft: 13,
+      },
+    }),
+  {
+    name: 'PostPreviewCard',
+  },
+);
+
+export interface IPostPreviewCardProps {
+  data: IPost;
+}
+
+const PostPreviewCard: React.FC<IPostPreviewCardProps> = (props) => {
+  const { data } = props;
+
+  const firstName = data.author?.firstName as string;
+  const secondName = data.author?.secondName as string;
```

Comment:
we don't need this type assertion.

create a custom logic in case there's no author provided, display an empty expert placeholder, for example

Reply:
fixed